src/towncrier/_settings.py

```diff
@@ -74,9 +75,12 @@ def load_config(directory):
 
 
 def load_config_from_file(directory, config_file):
-
-    with io.open(config_file, "r", encoding="utf8") as conffile:
-        config = _toml_load(conffile)
+    if tomli:
+        with open(config_file, "rb") as conffile:
```

Comment:
```suggestion
        with io.open(config_file, "rb") as conffile:
```

Reply:
This should be unnecessary in Python 3.6+ (which is always true in this branch)?

Reply:
yeah it's unnecessary, but I'm going to pyupgrade all these away in one go

Reply:
Haha, ok ill commit this if you want more work for pyupgrade? :smile: